docker/puppetserver/Dockerfile

```diff
@@ -8,7 +8,8 @@ ARG UBUNTU_CODENAME=bionic
 FROM ubuntu:18.04 as base
 
 ARG PACKAGES=ca-certificates\ git
-ARG DUMB_INIT_VERSION="1.2.2"
+ARG DUMB_INIT_VERSION="1.2.5"
+ARG TARGETARCH
```

Comment:
A comment here that this is set by `buildx` would be useful... it looks like magic because it's not passed in during the build